pkg/workspace/image/pusher.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+set -ex
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+TMPDIR="$(mktemp -d)"
+
+VOL_DIR="${1}"
+[ ! -z "${VOL_DIR}" ] || VOL_DIR='{{ .volDir }}'
+
+IMG_REF="${2}"
+[ ! -z "${IMG_REF}" ] || IMG_REF='{{ .imgRef }}'
+
+#{{`
+
+wait() {
+    until [ -e "${VOL_DIR}/fine_tuning_completed.txt" ]
```

Comment:
how about pass this sentinel file name through args? Determine it by upper control plane.
This script is expected to be simple and rarely updated.

Reply:
This was hard-coded in the previous script, so I thought it would be okay to do the same here.

Since I've written this in a way that allows local execution of the script, I can rewrite it in a way that uses the hard-coded values by default but allows these values to be changed when the script is rendered.